<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bgwidget</class>
 <widget class="QDialog" name="bgwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#bgwidget{
background-color:
 qlineargradient(spread:pad, x1:0.006, y1:0, x2:0.874, y2:0.927, stop:0 rgba(0, 170, 255, 255), stop:0.892045 rgba(216, 191, 191, 255));}</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Poppins&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Poppins&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Enter your email to change password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>600</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
font: 14pt &quot;Poppins&quot;;
background-color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>Proceed......</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updatePasswordField">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
font: 12pt &quot;Poppins&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="updateEmail">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:11pt &quot;Poppins&quot;;</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="updatePassword">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>420</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:11pt &quot;Poppins&quot;;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updateEmailField">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
font: 12pt &quot;Poppins&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="updateErrorMessage">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;Poppins&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="updateBackToLogin">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>680</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);
font: 14pt &quot;Poppins&quot;;
border-radius:5px</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>690</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Poppins&quot;;
</string>
   </property>
   <property name="text">
    <string>if you have an existing account</string>
   </property>
  </widget>
  <widget class="QLabel" name="updateEmailErrorMessage">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;Poppins&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="updateConPassword">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>500</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:11pt &quot;Poppins&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="updateConPasswordField">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>520</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
font: 12pt &quot;Poppins&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="updateBackToSignup">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>761</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);
font: 10pt &quot;Poppins&quot;;
border-radius:5px</string>
   </property>
   <property name="text">
    <string>Create an acccount</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
